App/src/main/java/com/google/sps/servlets/User.java

```diff
@@ -0,0 +1,144 @@
+package com.google.sps.servlets;
+
+public class User {
+	/*For the User class the fields needed are:
```

Comment:
Nitpick: Try to keep the formatting consistent with the comments if you can (keep the *s on the same line, Make the /* and */ match, etc.